opm/autodiff/BlackoilSolventModel_impl.hpp

```diff
@@ -284,6 +298,142 @@ namespace Opm {
         }
     }
 
+    template <class Grid>
+    void BlackoilSolventModel<Grid>::computeWellConnectionPressures(const SolutionState& state,
+                                                                        const WellState& xw)
+    {
+        if( ! Base::localWellsActive() ) return ;
```

Comment:
It is not possible to easily tell what is the difference between this and the base version by looking at the diff, so can you make a list of what is different?